src/models/kernels.hh

```diff
@@ -221,19 +220,14 @@ namespace rascal {
       using internal::TargetType;
 
       switch (this->target_type) {
-      case TargetType::Structure: {
+      case TargetType::Structure:
         return this->compute_helper<Property_t, TargetType::Structure>(
             representation_name, managers_a, managers_b);
-        break;
-      }
-      case TargetType::Atom: {
+      case TargetType::Atom:
         return this->compute_helper<Property_t, TargetType::Atom>(
             representation_name, managers_a, managers_b);
-        break;
-      }
       default:
         throw std::logic_error("The combination of parameter is not handdled.");
```

Comment:
Let's fix the spelling while we're at it:
```suggestion
        throw std::logic_error("The combination of parameter is not handled.");
```
Perhaps you could do a global `s/handdled/handled/`, since I think this line has been copy-pasted around quite a bit. But furthermore, it's an error message that should never be reached, so its utility is questionable -- I think this should be more along the lines of https://xkcd.com/2200/ -- implemented e.g. here: https://github.com/cosmo-epfl/librascal/blob/deb7a09bed3e8713267a364cd742687eb28aaa8c/src/representations/calculator_spherical_expansion.hh#L1028-L1039

Reply:
Looks like it is only written `handdled` in this file. 

> But furthermore, it's an error message that should never be reached, so its utility is questionable

We could have an unreachable macro/function for this, to be used everywhere. But I'd rather leave this for a future PR. Do you want me to open an issue?

Reply:
Yeah why not

Reply:
see #185 for a general fix to this.